client.go

```diff
@@ -0,0 +1,593 @@
+package milter
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"io"
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/emersion/go-message/textproto"
+)
+
+// Client is a wrapper for managing milter connections.
+//
+// Currently it just creates new connections using provided Dialer.
+type Client struct {
+	// Dialer is used to establish new connections to the milter.
+	// Set to empty net.Dialer{} by NewClient.
```

Comment:
This sentence should get removed, `NewClient` sets a non-zero timeout.

Reply:
Fixed in 563ba50.